lua/project_nvim/project.lua

```diff
@@ -254,8 +261,9 @@ function M.init()
     command! ProjectRoot lua require("project_nvim.project").on_buf_enter()
   ]])
 
-  autocmds[#autocmds + 1] =
-    'autocmd VimLeavePre * lua require("project_nvim.utils.history").write_projects_to_history()'
+  vim.cmd("command! ProjectAddMuanually lua _ADD_CURR_DIR_TO_PROJECTS()")
```

Comment:
I think there is a small typo here:

```suggestion
  vim.cmd("command! ProjectAddManually lua _ADD_CURR_DIR_TO_PROJECTS()")
```